src/app/shared/index.ts

```diff
@@ -0,0 +1,3 @@
+import { Route } from "./interfaces.routes";
+
+export { Route };
```

Comment:
```suggestion
export { Route }  from "./interfaces.routes";
```

You can use a direct export instead of import and export at the same time.